Ag changes

akeller opened the pull request:
* Updated banner with triangle asset
* Updated button color & hover

menski commented:
FYI the change "glossary updates for process terminology" (9a54a84) was already merge with PR #190

akeller commented:
> FYI the change "glossary updates for process terminology" ([9a54a84](https://github.com/camunda-cloud/camunda-cloud-documentation/commit/9a54a84c4e775bffd8489ebeaee669a111df8c68)) was already merge with PR #190

I removed it with a rebase. I also modified this PR to merge into `master` so we can push changes tomorrow.

menski reviewed (approved):
lgtm

pihme commented on docs/product-manuals/zeebe/open-source/deprecated-features.md:
```diff
@@ -32,3 +32,7 @@ In terms of specifying values, there were two minor changes:
 - Memory sizes are now specified like this: `512MB` (old way: `512M`)
 - Durations, e.g. timeouts, can now also be given in ISO-8601 Durations format. However you can still use the established way and specify a timeout of `30s`
+
+## Removed in 1.0
```
Being a bit nit-picky here: Since 1.0 is not released yet this currently describes the future. So we could either remove, or rephrase it to make it clearer that it will be removed.

pihme commented on docs/product-manuals/clients/cli-client/index.md:
```diff
@@ -54,7 +54,7 @@ Available Commands:
   complete    Complete a resource
   create      Create resources
-  deploy      Creates new workflow defined by provided BPMN or YAML file as workflowPath
+  deploy      Creates new workflow defined by provided BPMN file as workflowPath
```
As far as I know there is a deliberate renaming under way which replaces all instances of `workflow` with `process`. I am not sure whether this applies only to the code or also to the documentation.

See e.g. https://github.com/zeebe-io/zeebe/pull/6455

@menski can say more about this and give some context

pihme commented on docs/reference/grpc.md:
```diff
@@ -287,14 +287,16 @@ message WorkflowRequestObject {
     FILE = 0;
     BPMN = 1; // extension 'bpmn'
-    YAML = 2; // extension 'yaml'
+    YAML = 2 [deprecated = true]; // extension 'yaml'; removed as of release 1.0
```
again, not released yet

pihme commented on docs/reference/grpc.md:
```diff
@@ -287,14 +287,16 @@ message WorkflowRequestObject {
   // the resource type; if set to BPMN or YAML then the file extension
   // is ignored
-  ResourceType type = 2;
+  // As of release 1.0, YAML support was removed and BPMN is the only supported resource type.
```
again, not released yet

pihme commented on versioned_docs/version-0.26/guides/getting-started/connect-to-your-cluster.md:
````diff
@@ -0,0 +1,38 @@
+Install the appropriate package.
+
+```bash
+npm i -g zbctl
````
We had a discussion internally whether this is the best installation source. I think the NPM package is maintained by Josh and not up to date all the time. On the plus side it is a very easy installation.
Our internal discussion didn't yield any clear preference, so it might be up for product management to decide whether they want to elevate the NPM package to officially supported or reduce its usage,

pihme reviewed (approved):
Some comments to consider. I didn't look at all the files, though. was a bit overwhelming.

akeller commented:
@pihme thanks for your comments. When I opened the PR I noticed these came over, possibly because I was originally merging into 'release_0_26'. Since these are not my changes and I'd like to merge this PR today I can remove the ones that are not mine and referenced in the description.

menski reviewed (approved):
:shipit:
